backend/infrahub/events/schema_action.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from infrahub.message_bus import InfrahubMessage
+from infrahub.message_bus.messages.refresh_registry_branches import RefreshRegistryBranches
+
+from .models import InfrahubBranchEvent
+
+
+class SchemaUpdatedEvent(InfrahubBranchEvent):
```

Comment:
i think it would be good to add the name of the branch that was updated and if possible the new value of the hash for this branch

Reply:
The branch name is already included although it comes from the `InfrahubBranchEvent` object (not sure what I think about these nested object structures..).

I'll add the schema_hash, and add it to the payload at least. The get_resource method only expects a dictionary with two keys is that correct? 

```python
    def get_resource(self) -> dict[str, str]:
        return {
            "prefect.resource.id": f"infrahub.schema_branch.{self.branch}",
            "infrahub.branch.name": self.branch,
        }
```

Reply:
it expects one dict but it can have as many entries as necessary

Reply:
Ok, thanks!